msyncd/AccountsHelper.cpp

```diff
@@ -186,7 +189,7 @@ void AccountsHelper::slotAccountEnabledChanged(const QString &serviceName, bool
             foreach(SyncProfile *profile, profiles)
             {
                 // See if the service name matches
-                if(serviceName == profile->name())
+                if(serviceName == profile->key(REMOTE_SERVICE_NAME))
                 {
```

Comment:
I agree - but why was this if-check there originally?  Who added it, and what did they state was the reason for doing so?  Or is it legacy from harmattan-era with no purpose?